bin/idl.js

```diff
@@ -534,6 +534,17 @@ function fetch(service, cb) {
         }
     }
 
+    function findService(json, service) {
+        var path = service.split('/');
```

Comment:
path is used for directory/file paths. This is the path stored in the meta.json and need not map directly to a folder structure.